src/components/Dialog/index.test.tsx

```diff
@@ -32,9 +33,9 @@ describe('Dialog', () => {
 
     /* Tests that onDismiss gets fired correctly */
     /* ========================================= */
-    fireEvent.click(screen.getByLabelText('Close'));
+    userEvent.click(screen.getByLabelText('Close'));
     // Pressing esc anywhere in dialog should trigger a close
-    fireEvent.keyDown(screen.getByText('content'), { key: 'Escape', code: 27 });
+    userEvent.keyboard('{Escape}');
```

Comment:
lol this is why I made this ticket. I was like "wait... this would be so much more easier with `userEvent` when you made that Dialog PR" but then I realized you were just using what was there. So that's the motivator behind why I made this ticket.

Reply:
lol ya makes sense